app/controllers/api/api_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::ApiController < ActionController::Base
```

Comment:
Good to see a dedicated parent class for the ApiControllers, stops the ApplicationController from getting over burdened with responsibilities. Well done.